tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/TypeSpec/TspConvertTool.cs

```diff
@@ -162,7 +168,7 @@ CancellationToken ct
             return null; // Validation passed
```

Comment:
I can't leave a comment on the right spot, but above this you've just started your newline additions again. AGAIN! :)